x/concentrated-liquidity/module.go

```diff
@@ -63,24 +66,27 @@ func (b AppModuleBasic) GetQueryCmd() *cobra.Command {
 // RegisterInterfaces registers interfaces and implementations of the gamm module.
 func (AppModuleBasic) RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	types.RegisterInterfaces(registry)
+	clmodel.RegisterInterfaces(registry)
 }
 
 type AppModule struct {
 	AppModuleBasic
 
-	k Keeper
+	keeper Keeper
 }
 
-func (am AppModule) RegisterServices(cfg module.Configurator) {
-}
-
-func NewAppModule(concentratedLiquidityKeeper Keeper) AppModule {
+func NewAppModule(cdc codec.Codec, keeper Keeper) AppModule {
 	return AppModule{
-		AppModuleBasic: AppModuleBasic{},
-		k:              concentratedLiquidityKeeper,
+		AppModuleBasic: AppModuleBasic{cdc: cdc},
+		keeper:         keeper,
 	}
 }
 
+func (am AppModule) RegisterServices(cfg module.Configurator) {
+	types.RegisterMsgServer(cfg.MsgServer(), NewMsgServerImpl(&am.keeper))
+	clmodel.RegisterMsgCreatorServer(cfg.MsgServer(), NewMsgCreatorServerImpl(&am.keeper))
+}
+
```

Comment:
I think I did this because model used to be in the internal package, but it no longer is, so these should be able to be combined. Will do that now, thanks for pointing this out!

Reply:
@p0mvn is there a reason you dont initialize those other message servers here though? https://github.com/osmosis-labs/osmosis/blob/5f4bafc9c6de332f598daa6460975f8f32d1fc79/x/gamm/module.go#L105-L111